common-content/modules/ROOT/partials/get-started.adoc

```diff
@@ -33,16 +33,16 @@ The following languages and frameworks are officially supported by Neo4j:
 | https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md[.NET Standard 2.0]
 
 | Go
-| All the Go versions officially supported, with 1.18 as a minimum (https://go.dev/doc/devel/release#policy)
+| All officially maintaned Go versions, with 1.18 as a minimum (https://go.dev/doc/devel/release#policy).
```

Comment:
```suggestion
| All officially maintained Go versions, with 1.18 as a minimum (https://go.dev/doc/devel/release#policy).
```

Reply:
Note/nit: you could drop the `with 1.18 as a minimum` since that's included in the supported versions described in https://go.dev/doc/devel/release#policy

Reply:
Thanks. I left the `1.18` mention just cause it recently popped up and I thought it needed to be. Removed now.